### PR TITLE
Hamamatsu - LimaDetector: If Specific is in FAULT state during intialization the LimaDetector is in FAUTL state too

### PR DESCRIPTION
**Pull request by sbahji**
[ICATHALES-464][Hamamatsu] If Specific is FAULT during init_device the LimaDetector

### CODE REVIEW

File: specifics/Hamamatsu/Hamamatsu.cpp [hunk 1]
```diff
@@ -640,10 +640,13 @@ void Hamamatsu::always_executed_hook()
 {
     try
     {
-        m_status_message.str("");
-
         yat::AutoMutex<> _lock(ControlFactory::instance().get_global_mutex());
 
+        if(m_is_device_initialized)
```
Comment (soleil-ica) [inline specifics/Hamamatsu/Hamamatsu.cpp:645]: ajouter un commentaire

File: src/Factory.cpp [hunk 2]
```diff
@@ -1358,35 +1358,38 @@ Tango::DevState ControlFactory::get_state(void)
     CtControl::Status ctStatus;
     m_control->getStatus(ctStatus);
 
-    switch(ctStatus.AcquisitionStatus)
+    if(m_state != Tango::FAULT)
```
Comment (soleil-ica) [inline src/Factory.cpp:1361]: ajouter un commentaire: pourquoi si state != Fault?

**Review by soleil-ica — CHANGES_REQUESTED** [review]

File: specifics/Hamamatsu/Hamamatsu.cpp (hunk 1 above)
Comment (sbahji) [reply specifics/Hamamatsu/Hamamatsu.cpp:646]: ok

File: src/Factory.cpp (hunk 2 above)
Comment (sbahji) [reply src/Factory.cpp:1362]: ok

**Review by soleil-ica — APPROVED** [review]

**Pull request merged**